packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -50,28 +50,27 @@ export const messages = {
     'Enter desired directory (Press Enter to confirm or Esc to cancel)',
   parameter_gatherer_enter_username_name: 'Enter target username',
   parameter_gatherer_enter_alias_name:
-    'Enter a scratch org alias or use default alias',
+    'Enter an org alias or use default alias',
   parameter_gatherer_enter_project_name: 'Enter project name',
   parameter_gatherer_paste_forceide_url: 'Paste forceide:// URL from Setup',
   parameter_gatherer_paste_forceide_url_placeholder:
     'forceide:// URL from Setup',
   parameter_gatherer_invalid_forceide_url:
     "The forceide:// URL is invalid. From your subscriber's org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.",
 
-  force_org_create_default_scratch_org_text:
-    'SFDX: Create a Default Scratch Org...',
+  force_org_create_default_scratch_org_text: 'SFDX: Create a Default Org...',
```

Comment:
This command only works to create scratch orgs, so I'd leave it as is.

Reply:
Fixing this as well.